src/win/process.c

```diff
@@ -375,6 +423,9 @@ static WCHAR* search_path(const WCHAR *file,
   /* Check if the filename includes an extension */
   dot = wcschr(file_name_start, L'.');
   name_has_ext = (dot != NULL && dot[1] != L'\0');
+  if (name_has_ext) {
+    *isScript = (_wcsicmp(dot, L".bat") == 0 )||(_wcsicmp(dot, L".cmd") == 0);
```

Comment:
* Style: add spaces around the `||`.
* I think this would trigger a false positive if the file name had .bat somewhere int he path, e.g. `c:\foo\bert.bat\run.exe`.